### ISSUE TITLE
Enable automatic detection of Zotero and PDF files.

### ISSUE THREAD

**Issue by burtonator** [burtonator/polar-bookshelf#631]
Automatically download files from the Zotero directory:

https://www.zotero.org/support/zotero_data

![image](https://user-images.githubusercontent.com/45447/53115194-6dcb0200-34fa-11e9-8fb3-5f40b54f89f7.png)

Also consider automatically importing from OS specific Downloads and Documents folders.

**Comment by danieltomasz** [comment]
Zotero is the folder with user settings ( it is usually containing folder `storage` where Zotero hold files,  I am using Zotfiles and put my Zotero library  to Dropbox folder)
![schermafdruk_2019-02-22_17-01-28](https://user-images.githubusercontent.com/7980381/53254449-ac5fe880-36c3-11e9-95dc-7a63506f5e38.png)

**Comment by burtonator** [comment]
I'm not seeing a storage file.  Are the .pdfs there as just normal files?  

If so I can detect the directory and just auto-import.

**Comment by danieltomasz** [comment]
here is no `storage` folder because I change the default folder to point into my dropbox, not storage.
Pdfs are  normal files, but they are contained  within alphanumeric  subfolders (the subfolder also contain sometimes some additional  temp files  created by Zotero), now  I added new file  as example
![schermafdruk_2019-02-22_17-22-35](https://user-images.githubusercontent.com/7980381/53255768-9e5f9700-36c6-11e9-96d6-6ca0a080b807.png)

Many people who use plugin `Zotfile` may have different structure and different location of the pdf folder (for example subfolders for authors)

**Comment by burtonator** [comment]
Is it a subfolder of the main zotero dir.? If so it should be easy for me to find it. In your situation seems like you have a custom path.

**Comment by danieltomasz** [comment]
Yes, by default  Zotero uses paths from your first post from this issue (and store pdf in `storage` folder), I changed the path - maybe option to specify Zotero path for the user themselves would be enough. Maybe other users could say more https://github.com/burtonator/polar-bookshelf/issues/594
